docs/source/changelog.rst

```diff
@@ -1,6 +1,23 @@
 Changelog
 =========
 
+0.17.X / 2018-02-DD
+-------------------
+
+Array
++++++
+
+DataFrame
++++++++++
+
+- Add ``dd.tseries.Resampler.agg`` (:pr:`3202`) `Richard Postelnik`_
+
+Bag
++++
+
+Core
+++++
+
```

Comment:
Thank you for setting this up!  My bad on omitting it when doing the last release.